sqlalchemy_bigquery/_helpers.py

```diff
@@ -24,19 +25,46 @@
 )
 
 
-def google_client_info():
-    user_agent = USER_AGENT_TEMPLATE.format(sqlalchemy.__version__)
+def google_client_info(user_agent=None):
```

Comment:
NIT:

Consider this isn's a private method, maybe we can add type hint for this parameter too?

Reply:
Done.